Added Kinect rules, modprobe blacklist and fixed the owner in the PrimeSense rules.

From: Roman Donchenko <roman.donchenko@itseez.com>

The Kinect rules are from <http://github.com/avin2/SensorKinect/>.
---
 Platform/Linux/CreateRedist/install.sh         |    6 ++++++
 Platform/Linux/Install/55-primesense-usb.rules |   19 +++++++++++++------
 2 files changed, 19 insertions(+), 6 deletions(-)

diff --git a/Platform/Linux/CreateRedist/install.sh b/Platform/Linux/CreateRedist/install.sh
index 2a28463..2217dad 100644
--- a/Platform/Linux/CreateRedist/install.sh
+++ b/Platform/Linux/CreateRedist/install.sh
@@ -74,6 +74,7 @@ INSTALL_LIB=$rootfs/usr/lib
 INSTALL_BIN=$rootfs/usr/bin
 INSTALL_ETC=$rootfs/usr/etc/primesense
 INSTALL_RULES=$rootfs/etc/udev/rules.d
+INSTALL_BLACKLIST=$rootfs/etc/modprobe.d
 SERVER_LOGS_DIR=$rootfs/var/log/primesense/XnSensorServer
 
 # make all calls into OpenNI run in this filesystem
@@ -130,6 +131,11 @@ if [ "$install" = yes ]; then
         printf "installing usb rules..."
         cp Install/$RULES_FILE $INSTALL_RULES
         printf "OK\n"
+
+        printf "installing modprobe blacklist..."
+        mkdir -p $INSTALL_BLACKLIST
+        echo 'blacklist gspca_kinect' > $INSTALL_BLACKLIST/blacklist-gspca-kinect.conf
+        printf "OK\n"
     fi
 
 	printf "\n*** DONE ***\n\n"
diff --git a/Platform/Linux/Install/55-primesense-usb.rules b/Platform/Linux/Install/55-primesense-usb.rules
index 6294f40..621fe3b 100644
--- a/Platform/Linux/Install/55-primesense-usb.rules
+++ b/Platform/Linux/Install/55-primesense-usb.rules
@@ -1,8 +1,15 @@
 # make primesense device mount with writing permissions (default is read only for unknown devices)
-SYSFS{idProduct}=="0200",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0300",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0400",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0500",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0600",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
-SYSFS{idProduct}=="0601",SYSFS{idVendor}=="1d27",MODE="666",OWNER="xxx",GROUP="users"
+SYSFS{idProduct}=="0200",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0300",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0400",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0500",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0600",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
+SYSFS{idProduct}=="0601",SYSFS{idVendor}=="1d27",MODE="666",OWNER="root",GROUP="users"
 
+#--avin mod--
+# Kinect
+SUBSYSTEM=="usb", ATTR{idProduct}=="02ae", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02ad", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="audio"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02b0", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02be", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="audio"
+SUBSYSTEM=="usb", ATTR{idProduct}=="02bf", ATTR{idVendor}=="045e", MODE:="0666", OWNER:="root", GROUP:="video"
